staging/src/k8s.io/apiserver/pkg/storage/cacher/cacher.go

```diff
@@ -280,7 +280,7 @@ type Cacher struct {
 
 	// indexedTrigger is used for optimizing amount of watchers that needs to process
 	// an incoming event.
-	indexedTrigger *indexedTriggerFunc
+	indexedTrigger map[string]*indexedTriggerFunc
```

Comment:
If you're going to change this, please extract the changes done to cacher to a separate PR.
And you will for sure need a pretty extensive testing of that.

Reply:
I removed the changes in cacher.go. Thank you!